scripts/__init__.py

```diff
@@ -0,0 +1 @@
+from scripts.cmfgen2tardis import *
```

Comment:
@wkerzendorf @yeganer When I leave `__init__.py` file empty, travis fails, because it skips this file.
(`warning: build_scripts: scripts/__init__.py is an empty file (skipping)`). [Travis Report](https://travis-ci.org/tardis-sn/tardis/jobs/255615422).
Can this be avoided.